kyo-scheduler/jvm-native/src/main/scala/kyo/scheduler/Worker.scala

```diff
@@ -92,7 +93,8 @@ abstract private class Worker(
 
     val a1, a2, a3, a4, a5, a6, a7 = 0L // padding
 
-    @volatile private var state: State  = State.Idle
+    private val state = new AtomicReference[State](State.Idle)
```

Comment:
Scala Native doesn't support method handles. This change might have a performance penalty but it seems worth it.